torchbenchmark/util/framework/vision/model_factory.py

```diff
@@ -25,7 +26,11 @@ def __init__(self, model_name, test, device, jit=False, batch_size=None, extra_a
             self.example_outputs = torch.rand_like(self.model(*self.example_inputs))
             self.model.train()
             # setup optimizer and loss_fn
-            self.optimizer = optim.Adam(self.model.parameters())
+            self.optimizer = optim.Adam(
```

Comment:
Will this break the benchmarks when running with older pytorch builds? When was the `capturable=` argument introduced?

Reply:
@xuzhao9 it was added in june 2022 (https://github.com/pytorch/pytorch/pull/77862), is that old enough? Otherwise I think it's okay to remove it

Reply:
Also could you explain the reason to be concerned about running with older pytorch builds, and what the guidelines are for how long the feature should be in pytorch before it's safe to include in a benchmark here? Since these tests are targeted for ddp testing, we'll probably eventually start adding some features that depend on new dynamo features.

Reply:
I  am asking this because we sometimes receive GitHub issues of users complaining the benchmark doesn't run on the latest stable release (such as https://github.com/pytorch/benchmark/issues/405) and we have to add bc-compatible layers to handle it.